tests/phpunit/integration/Core/User/REST_Audience_Settings_ControllerTest.php

```diff
@@ -79,21 +80,7 @@ public function test_register() {
 		$this->assertTrue( has_filter( 'googlesitekit_apifetch_preload_paths' ) );
 	}
 
-	public function test_get_routes__no_feature_flag() {
-		$this->controller->register();
-
-		$server = rest_get_server();
-		$routes = array(
-			'/' . REST_Routes::REST_ROOT . '/core/user/data/audience-settings',
-
-		);
-		$get_routes = array_intersect( $routes, array_keys( $server->get_routes() ) );
-
-		$this->assertTrue( empty( $get_routes ) );
-	}
-
-	public function test_get_routes__with_feature_flag() {
-		$this->enable_feature( 'audienceSegmentation' );
+	public function test_get_routes__audience_settings_available() {
```

Comment:
We can just call this `test_get_routes()`:

```suggestion
	public function test_get_routes() {
```